boolean-2.js

```diff
@@ -0,0 +1,28 @@
+/* Create a function close_compare that accepts 3 parameters: a, b, and an optional margin. The function should return whether a is lower than, close to, or higher than b.
+
+Please note the following:
+
+When a is close to b, return 0.
+For this challenge, a is considered "close to" b if margin is greater than or equal to the absolute distance between a and b.
+Otherwise...
+
+When a is less than b, return -1.
+
+When a is greater than b, return 1.
+
+If margin is not given, treat it as if it were zero.
+
+Assume: margin >= 0
+
+Tip: Some languages have a way to make parameters optional.
+*/
+
+/*
+Example 1
+If a = 3, b = 5, and margin = 3, then close_compare(a, b, margin) should return 0.
+
+This is because a and b are no more than 3 numbers apart.
+
+*/
+
+closeCompare = (a, b, margin) => Math.abs(a - b) <= margin ? 0 : Math.sign(a - b)
```

Comment:
skad masz to rozwiazanie? nie przypomina tego co mamy na zajeciach 🤔

Reply:
W rozwiązaniu pomógł internet, rzeczywiście jest to zaawansowana propozycja, wrzuciłem bardziej "moją", wcześniej mi się ni udawało, teraz działa.